### DISCUSSION TITLE
新手绳匠！

### DISCUSSION THREAD

**Discussion by Nyanonako** [share121/inter-knot#1581] [其他]
刚注册用户，请问要注意什么吗？

**Comment by gaojiaobei** [comment]
建议把委托留给老用户们，减缓接委托的速度
因为初来乍到各种方面都不成熟，少做少错，多看看老前辈们的行动过程，多多学习
当然，最主要的是我们这种老绳匠有更多丁尼赚

**Comment by ZhuYuan-148** [comment]
注意不要做违法犯罪行为。空洞内不扰乱正常秩序的话我也是睁一只眼闭一只眼的。
